src/runtime/BigInt.h

```diff
@@ -27,13 +27,22 @@ namespace Escargot {
 class BigInt : public PointerValue {
 public:
     BigInt(VMInstance* vmInstance, int64_t num);
+    static Optional<BigInt*> parseString(VMInstance* vmInstance, const char* buf, size_t length, int radix = 10);
+    static Optional<BigInt*> parseString(VMInstance* vmInstance, String* str, int radix = 10);
```

Comment:
Its my opinion about function naming.
I think changing these function's name to `convertStringToBigInt` or something similar to this would be better to clearly represent the functions.
(you can ignore it if you don't want)

Reply:
IMO parsing string for producing BigInt is better than `convert...`

Reply:
Ok no problem :)